### DISCUSSION TITLE
Virtualized columns

### DISCUSSION THREAD

**Discussion by otariterterashvili** [nick-keller/react-datasheet-grid#132] [Feature requests]
In my Application, where I am using this Component, I have 100 columns, So thinks it's good idea to add Virtualized columns same as rows.

**Comment by nick-keller** [comment]
The lib I am using ([react-window](https://www.npmjs.com/package/react-window)) does not allow this. 
The tricky part is that I'm using sticky header and sticky columns left and right, which is not possible with a Grid, so I am using a list.
The solution would be to implement a custom virtualization solution, but that is a lot of work

**Reply by otariterterashvili** [reply]
I will dig into.
Thanks for response.

**Reply by alkima5** [reply]
any update on this? how can i do virtualize columns? i have 500 cols and 10k rows

**Reply by nick-keller** [reply]
No, my answer is still relevant. The only option would be to change virtualization lib, which is a lot of work.

**Comment by nick-keller** [comment]
@otariterterashvili @alkima5 
Virtualized columns are now available in 4.6.0
